flax/version.py

```diff
@@ -13,5 +13,5 @@
 # limitations under the License.
 
 """Current Flax version at head on Github."""
-__version__ = "0.6.7"
+__version__ = "0.6.8"
```

Comment:
This will result in users getting `0.6.8` if they do:

```python
import flax
print(flax.__version__)
```
I think we have to update this _after_ release to keep it on ahead.

Reply:
Ah, this makes sense!